### PR TITLE
Wheel circumference settings

### PR DESCRIPTION
**Pull request by DmitrySvetlichny**

### CODE REVIEW

File: Sources/Plugins/ExternalSensorsPlugin/BLE/BLEManager/Sensors/BLEBikeSensor.swift [hunk 1]
```diff
@@ -24,7 +24,15 @@ final class BLEBikeSensor: Sensor {
     var lastBikeCadenceData: BikeCadenceData?
     var lastBikeSpeedDistanceData: BikeSpeedDistanceData?
     // NOTE: wheelCircumference = wheelSize * pi
-    var wheelSize: Double = 2.086
+    var wheelSize: Double {
```
Comment (alex-osm) [inline Sources/Plugins/ExternalSensorsPlugin/BLE/BLEManager/Sensors/BLEBikeSensor.swift:27]: Sensor class represents hardware sensor (reading/writing from/to device), it is not related to app settings in any way

File: Sources/Plugins/ExternalSensorsPlugin/BLE/BLEWheelSettings/BLEWheelSettingsViewController.swift [hunk 2]
```diff
@@ -46,10 +48,13 @@ final class BLEWheelSettingsViewController: OABaseNavbarViewController {
     
     override func onRightNavbarButtonPressed() {
         if wheelSizeString != String(wheelSize) {
-            // TODO: need ui design
-          //  device.deviceName = wheelSizeString
-           // DeviceHelper.shared.changeDeviceName(with: device.id, name: newDeviceName)
-            onSaveAction?()
+            if let millimeters = Float(wheelSizeString) {
+                let meters = millimeters / 1000.0
```
Comment (alex-osm) [inline Sources/Plugins/ExternalSensorsPlugin/BLE/BLEWheelSettings/BLEWheelSettingsViewController.swift:52]: Is there any sense to store wheel size in meters?

File: Sources/Plugins/ExternalSensorsPlugin/BLE/DeviceSettings/DevicesSettingsCollection.swift [hunk 3]
```diff
@@ -79,6 +78,20 @@ final class DevicesSettingsCollection {
         }
     }
     
+    func changeWheelSize(with id: String, size: Float) {
```
Comment (alex-osm) [inline Sources/Plugins/ExternalSensorsPlugin/BLE/DeviceSettings/DevicesSettingsCollection.swift:81]: 1. There should be more common method like `changeDeviceParameter` (device id, param id, value).
2. Use `setDeviceProperty` for settings parameter

File: Sources/Plugins/ExternalSensorsPlugin/BLE/DeviceHelper.swift [hunk 4]
```diff
@@ -110,6 +110,10 @@ final class DeviceHelper: NSObject {
         devicesSettingsCollection.changeDeviceName(with: id, name: name)
     }
     
+    func changeWheelSize(with id: String, size: Float) {
```
Comment (alex-osm) [inline Sources/Plugins/ExternalSensorsPlugin/BLE/DeviceHelper.swift:113]: `changeDeviceParameter`

**Review by tigrim — APPROVED** [review]

**Pull request merged**